github/compliance/model.py

```diff
@@ -79,13 +81,17 @@ class ScanState(enum.Enum):
 
 @dataclasses.dataclass(kw_only=True)
 class ScanResult:
-    scanned_element: Target
+    scanned_element: (
+        cnudie.iter.ResourceNode
```

Comment:
why? we defined the type-alias `Target` precisely to abstract from the concrete types; which allows us to also extend the allowed types later. by resolving it here, this adds one piece of code that will not be found e.g. by searching for this type-alias.